### PR TITLE
Changes for liquidation system

### PR DESCRIPTION
**Pull request by HubertBubert**
Ready for review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Pull requests containing bugfixes are very welcome!
 
 ## License
 
-Copyright &copy; 2017 Quedex Ltd. API is released under [Apache License Version 2.0](LICENSE).
+Copyright &copy; 2019 Quedex Ltd. API is released under [Apache License Version 2.0](LICENSE).
```
Comment (mchalapuk) [inline README.md]: It sholud be 2017 - 2019. And you could change copyright notice in LICENSE file, if you're changing this.

File: src/main/java/net/quedex/api/market/MarketMessageReceiver.java [hunk 2]
```diff
@@ -126,6 +140,9 @@ protected void processData(String data) throws IOException, PGPExceptionBase {
                 break;
             case "instrument_data":
                 onInstrumentData(OBJECT_MAPPER.treeToValue(dataJson.get("data"), InstrumentsMap.class));
+                break;
+            case "spot_data":
+                onSpotDataWrapper(OBJECT_MAPPER.treeToValue(dataJson, SpotDataWrapper.class));
```
Comment (mchalapuk) [inline src/main/java/net/quedex/api/market/MarketMessageReceiver.java]: missing break

File: README.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ Include in your project as a Maven dependency:
 <dependency>
     <groupId>net.quedex</groupId>
     <artifactId>java-api</artifactId>
-    <version>0.7.0</version>
+    <version>0.8.0</version>
```
Comment (wgromniak) [inline README.md]: I'd first publish, then change version here.

File: src/main/java/net/quedex/api/market/SpotDataWrapper.java [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class SpotDataWrapper {
+
+    // maps from underlying symbol to SpotDataWrapper.
```
Comment (wgromniak) [inline src/main/java/net/quedex/api/market/SpotDataWrapper.java]: Comment not correct?

File: README.md (hunk 3 above)
Comment (mchalapuk) [reply README.md]: Shouldn't it be 0.8.0-SNAPSHOT for now?

**Pull request merged**